src/Classes/Options/Interactivity/Events/Events.ts

```diff
@@ -0,0 +1,72 @@
+import {IEvents} from "../../../../Interfaces/Options/Interactivity/Events/IEvents";
+import {ClickEvent} from "./ClickEvent";
+import {IDivEvent} from "../../../../Interfaces/Options/Interactivity/Events/IDivEvent";
+import {IHoverEvent} from "../../../../Interfaces/Options/Interactivity/Events/IHoverEvent";
+import {DivEvent} from "./DivEvent";
+import {HoverEvent} from "./HoverEvent";
+import {IClickEvent} from "../../../../Interfaces/Options/Interactivity/Events/IClickEvent";
+
+export class Events implements IEvents {
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new onClick
+     */
+    public get onclick(): IClickEvent {
+        return this.onClick;
+    }
+
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new onClick
+     * @param value
+     */
+    public set onclick(value) {
+        this.onClick = value;
+    }
+
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new onDiv
+     */
+    public get ondiv(): IDivEvent {
+        return this.onDiv;
+    }
+
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new onDiv
+     * @param value
+     */
+    public set ondiv(value) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected parameter: 'value' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5136254)